test/test-ce.html

```diff
@@ -2,13 +2,15 @@
 <html>
   <head>
     <title>Fragment test page CE</title>
+    <meta x-fragment headers-cache-control="must-revalidate" method="GET" mode="cors">
```

Comment:
Let's pass two params to headers
```headers-cache-control="must-revalidate"``` and
```headers-custom-header="custom-value"``` and see if they fall into attributes or not.